### ISSUE TITLE
When filtering on one project, also filter debug flags for that project

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#825]
**Reported by Ageless on 11 Jan 39147383 12:26 UTC**
When I am now filtering the Messages in BOINC Manager on one project, I only get the high level messages. The low level messages that I get with debug flags for that project go 'lost'. 

I'd like to request that these stay visible even when filtering on the one project. 

Example without filter:
```
22-Feb-09 14:01:14 SETI@home Started upload of 16ja09ab.5499.8252.9.8.247_1_0
22-Feb-09 14:01:14 [http://setiboincdata.ssl.berkeley.edu/sah_cgi/file_upload_handler
22-Feb-09 14:01:35 Project communication failed: attempting access to reference site
22-Feb-09 14:01:35 [file_xfer_debug](file_xfer_debug]URL:)FILE_XFER_SET::poll(): http op done; retval -107
22-Feb-09 14:01:35 [file_xfer_debug]file transfer status -107
22-Feb-09 14:01:35 SETI@home Temporarily failed upload of 16ja09ab.5499.8252.9.8.247_1_0: connect() failed
22-Feb-09 14:01:35 SETI@home Backing off 3 hr 7 min 42 sec on upload of 16ja09ab.5499.8252.9.8.247_1_0
22-Feb-09 14:01:36 Internet access OK - project servers may be temporarily down.
```

Example of that same bit with filter:
```
22-Feb-09 14:01:35 SETI@home Temporarily failed upload of 16ja09ab.5499.8252.9.8.247_1_0: connect() failed
22-Feb-09 14:01:35 SETI@home Backing off 3 hr 7 min 42 sec on upload of 16ja09ab.5499.8252.9.8.247_1_0
```

**Comment by romw** [comment]
**Commented by davea on 10 Jan 39148211 12:00 UTC**
The issue is that some messages (of the file_xfer_debug sort) aren't being associated with a project, when they could be.  I fixed this; there may be other cases.

**Comment by romw** [comment]
**Commented by davea on 7 Nov 39148211 10:13 UTC**
(In [17334]) - client: print message if downloaded file has wrong size
- client: associate file xfer messages with a project; fixes #848

**Comment by romw** [comment]
**Commented by Ageless on 14 Apr 39148392 04:53 UTC**
Thanks David. The only other one I see sometimes missing a project association is the sched_op_debug flag, on the handle_sched_reply().

**Comment by romw** [comment]
**Commented by Nicolas on 6 Oct 39148803 13:20 UTC**
I found some more, attaching patch against trunk.

**Comment by romw** [comment]
**Commented by Nicolas on 21 Oct 39148806 16:00 UTC**
I changed some result->project to wup->project and some 0s to NULLs just for consistency.

**Comment by romw** [comment]
**Commented by davea on 19 Aug 39149123 06:13 UTC**
(In [17335])  - client: more instances of showing project with message.  Fixes #848

**Comment by romw** [comment]
**Commented by romw on 23 Feb 39153939 06:40 UTC**
(In [17356])     - client: print message if downloaded file has wrong size
    - client: associate file xfer messages with a project; fixes #848

    client/
        cs_files.cpp
        file_xfer.cpp
        pers_file_xfer.cpp

**Comment by romw** [comment]
**Commented by romw on 7 Sep 39153964 23:33 UTC**
(In [17357])     - client: more instances of showing project with message.  Fixes #848

    client/
        app_start.cpp
        app_graphics.cpp
        file_xfer.cpp
        cs_account.cpp
        cs_scheduler.cpp
        scheduler_op.cpp
        app.cpp
        app_control.cpp
